skpy/chat.py

```diff
@@ -360,7 +360,8 @@ def rawToFields(cls, raw={}, active=False):
                        "open": props.get("joiningenabled", "") == "true",
                        "history": props.get("historydisclosed", "") == "true",
                        "picture": props.get("picture", "")[4:] or None,
-                       "active": active})
+                       "active": active,
+                       "moderatedthread": props.get("moderatedthread")})
```

Comment:
This isn't a boolean -- as it stands, the value will be `"true"` or `"false"`, or `None` on older chats that are missing this setting.  It should be checked for equality to `"true"`, as is done with the `open` and `history` attributes.